torch/distributions/utils.py

```diff
@@ -92,7 +92,7 @@ def probs_to_logits(probs, is_binary=False):
     return torch.log(ps_clamped)
 
 
-class lazy_property(object):
+class lazy_property:
```

Comment:
This transforms it into an old style class. We never want to use old style classes no?

Reply:
There is no such thing as an old-style class in Python 3, and PyTorch doesn't support Python 2, so writing `(object)` has no effect.

Reply:
Ho I missed that point when we moved out of Python 2!